### DISCUSSION TITLE
minio supports FTP upload file progress

### DISCUSSION THREAD

**Discussion by jiumengyixian** [minio/minio#20152] [Q&A] [answered]
When using minio as an FTP server, is it possible to get the progress of a file upload in real time when there is a file upload? Would like to get the progress from the minio-ftp side, not the upload side.

**Comment by harshavardhana** [comment]
Ftp server to persistent progress and progress seen on the client are same. It's stream to another stream.

There is no special difference here.

**Reply by jiumengyixian** [reply]
However, the question was asked based on the server-side perspective, if I am not a client, can I utilize the server-side to get the progress of the file upload.

**Reply by klauspost** [reply, marked as answer]
We have no plans to add that.

**Reply by jiumengyixian** [reply]
OK, thank you very much for your answers.
